votes/0701-xuyifangreeneyes-as-tidb-commiter.md

```diff
@@ -0,0 +1,23 @@
+# A Vote for xuyifangreeneyes as TiDB Committer
+
+## Proposal
+
+[@xuyifangreeneyes](https://github.com/xuyifangreeneyes) has contributed a lot to TiDB's planner.
+- He implemented the global kill for TiDB so that we can kill the process across the TiDB node, instead of just being able to kill the process of the corresponding TiDB node. 
```

Comment:
Global kill is implemented by @pingyu. I just work on some GA tests and global kill for auto analyze.